### PR TITLE
bugfix

### PR DESCRIPTION
**Pull request by taras-sereda**

### CODE REVIEW

**Comment by kylegao91** [comment]
Thanks!  Would you mind writing a small test case for it?

**Comment by taras-sereda** [comment]
I'm not sure this fix requires new test case. There was a wrong if statement. And next statement:
```kwargs['include_lengths'] = True``` would be executed anyway. It would be safer to set values inside if statements, and by doing this,  already present testCase would cover this issue.

**Comment by taras-sereda** [comment]
pushed new commit with safe assignment.

File: seq2seq/dataset/fields.py [hunk 1]
```diff
@@ -10,10 +10,10 @@ def __init__(self, **kwargs):
 
         if kwargs.get('batch_first') is False:
             logger.warning("Option batch_first has to be set to use pytorch-seq2seq.  Changed to True.")
-        kwargs['batch_first'] = True
-        if kwargs.get('batch_first') is False:
+            kwargs['batch_first'] = True
+        if kwargs.get('include_lengths') is False:
             logger.warning("Option include_lengths has to be set to use pytorch-seq2seq.  Changed to True.")
-        kwargs['include_lengths'] = True
```
Comment (kylegao91) [inline seq2seq/dataset/fields.py]: `kwargs['include_lengths']` has to be set and it has to be `True`.  Your change doesn't make sure that it has a value, which leads to the test failure.

Comment (taras-sereda) [reply seq2seq/dataset/fields.py]: oh, you are right

**Pull request merged**